Microsoft.Azure.Cosmos/src/Fluent/CosmosClientBuilder.cs

```diff
@@ -424,12 +424,30 @@ public CosmosClientBuilder WithConsistencyLevel(Cosmos.ConsistencyLevel consiste
         }
 
         /// <summary>
-        /// If Open Telemetry listener is subscribed for Azure.Cosmos namespace, There are <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> you can leverage to control it.<br></br>
+        /// Disables Operation level activity generation even if listener is subscribed to "Azure.Cosmos.Operation" source.
```

Comment:
Summary is no longer valid. Maybe something like (feel free to adjust):

```suggestion
        /// Sets whether Distributed Tracing for "Azure.Cosmos.Operation" source is enabled.
```